internal/bundle/testdata/fail_PolicyMissingChecksRuleID.yaml

```diff
@@ -0,0 +1,209 @@
+
+policies:
```

Comment:
Names should not be use different styles. Here we use snake casing with camel casing